include/uriparser/Uri.h

```diff
@@ -201,6 +201,15 @@ typedef struct URI_TYPE(QueryListStruct) {
 } URI_TYPE(QueryList); /**< @copydoc UriQueryListStructA */
 
 
+/**
+ * Checks if a URI has the host component set.
+ *
+ * @param uri %URI to check
```

Comment:
This line does not yet match the style of the rest of uriparser.

```suggestion
 * @param uri <b>IN</b>: %URI to check
```